frontend/src/components/page_content/configuration-content.js

```diff
@@ -1,9 +1,38 @@
 import React from "react";
+import Style from "style-it";
+
+import EmergencySavings from "../page_content/configurations/emergency-savings";
```

Comment:
the name is 'configuration' everywhere else so we should stick to 'configuration' and not 'configurations' so we don't mess it up somewhere